code/game/objects/items/cards_ids.dm

```diff
@@ -571,16 +571,6 @@ update_label()
 	access = list(ACCESS_SYNDICATE)
 	uses_overlays = FALSE
 
-/obj/item/card/id/syndicate_command/captain_id
```

Comment:
Did you mean to delete this?

Reply:
yeah, for some reason there's two syndi captain IDs with the same path, only difference being one didn't have much access